### PR TITLE
Bump  .NET framework target to net462

### PR DESCRIPTION
**Pull request by iamcarbon**
The final OS distribution supporting .NETFramework 4.5.2 has gone out of support. This bumps the library's minimum .NET Framework target to net 4.6.2.

### CODE REVIEW

**Comment by iamcarbon** [comment]
Ready for review.

File: src/Markdig/Helpers/ArrayHelper.cs [hunk 1]
```diff
@@ -12,15 +12,6 @@ public static class ArrayHelper
...
         /// </summary>
-#if NET452
-        public static T[] Empty<T>() => EmptyArray<T>.Value;
-
-        private static class EmptyArray<T>
-        {
-            public static readonly T[] Value = new T[0];
-        }
-#else
         public static T[] Empty<T>() => System.Array.Empty<T>();
```
Comment (MihaZupan) [inline src/Markdig/Helpers/ArrayHelper.cs:15]: I think we can remove this type now.

**Review by MihaZupan — APPROVED** [review]

**Comment by MihaZupan** [comment]
cc: @Kryptos-FR

File: src/Markdig/Helpers/ArrayHelper.cs (hunk 1 above)
Comment (iamcarbon) [reply src/Markdig/Helpers/ArrayHelper.cs]: done!

**Comment by xoofx** [comment]
Thanks!

**Pull request merged**